.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-24.04
     strategy:
       matrix:
-        python3-version: ['10', '11', '12', '13']
```

Comment:
Is dropping '13' here intentional for building wheels >=3.12?

Reply:
> Is dropping '13' here intentional for building wheels >=3.12?

Yes! We don't need to anymore, since the wheels built with 3.12 also work with 3.13.

Reply:
Oops yes ok I thought I'd misunderstood, thanks !